internal/literals/split.go

```diff
@@ -85,7 +87,7 @@ func (x split) obfuscate(data []byte) *ast.BlockStmt {
 	for i, index := range indexes[:len(indexes)-1] {
```

Comment:
The `decryptKeyInitial` in Line 82 is not used

Reply:
decryptKeyInitial used at 171 line

Reply:
Ok